File: src/Frontend/src/shared/models/character.model.ts
export class Character {
    id: string;
    class: string;
    name: string;
    level: number;
    isLadder: boolean;
    isExpansion: boolean;
    
    constructor(id: string, _class: string, name: string, level: number, isLadder: boolean, isExpansion: boolean) {
        this.id = id;
        this.class = _class;
        this.name = name;
        this.level = level;
        this.isLadder = isLadder;
        this.isExpansion = isExpansion;
     }
}

File: src/DiabloII-Cookbook.Client/Models/CharacterModel.cs
using DiabloII_Cookbook.Api.DataTransferObjects;
using System;
using System.Collections;
using System.Collections.Generic;

namespace DiabloII_Cookbook.Client.Models
{
    public class CharacterModel
    {
        public Guid Id { get; }
        public string Class { get; }
        public string Name { get; }
        public int Level { get; set; }
        public bool IsExpansion { get; }
        public bool IsLadder { get; }

        public CharacterModel(Guid id, string @class, string name, int level, bool isExpansion, bool isLadder)
        {
            Id = id;
            Class = @class;
            Name = name;
            Level = level;
            IsExpansion = isExpansion;
            IsLadder = isLadder;
        }
    }
}
